backend/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const app = express()
+app.use(express.json());
+const mongoose = require('mongoose')
+const connection_url = "mongodb+srv://newUser:newPassword@cluster0.a9qq6.mongodb.net/RecipesDB?retryWrites=true&w=majority"
+const recipeEndpoints = require("./routes/recipe")
+
+mongoose.connect(connection_url)
+    .then(() => console.log("Connection Success"))
+    .catch((error) => console.log("Could not Connect due to ${error}"))
```

Comment:
I believe that the error would not display as expected here. Make sure to use ` instead of " when we want to display a ${var} within a string.